app/src/main/res/drawable/context_menu_button_background.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+<shape
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:shape="oval">
+    <size
+        android:height="24dp"
+        android:width="24dp" />
+    <solid android:color="#e3e3e5" />
```

Comment:
Is this color something that will probably be reused? If so, pull it out into colors.xml.